pkg/kubelet/dockershim/libdocker/kube_docker_client.go

```diff
@@ -152,18 +143,18 @@ func (d *kubeDockerClient) CreateContainer(opts dockertypes.ContainerCreateConfi
 func (d *kubeDockerClient) StartContainer(id string) error {
 	ctx, cancel := d.getTimeoutContext()
 	defer cancel()
-	err := d.client.ContainerStart(ctx, id)
+	err := d.client.ContainerStart(ctx, id, dockertypes.ContainerStartOptions{})
 	if ctxErr := contextError(ctx); ctxErr != nil {
 		return ctxErr
 	}
 	return err
 }
 
 // Stopping an already stopped container will not cause an error in engine-v1.
```

Comment:
Changed the comment. I think the behavior stays the same since the implementations of the relevant functions do not change.